internal/preflight/preflight.go

```diff
@@ -195,3 +195,36 @@ func (p *preflightHelper) verifySign(ctx context.Context, pv *kmmv1beta1.Preflig
 	}
 	return false, "Waiting for sign verification"
 }
+
+func DeleteModuleStatus(statuses *[]v1beta2.PreflightValidationModuleStatus, nsn types.NamespacedName) {
```

Comment:
I see that thisfunction are using only in tests  -what am i missing?

Reply:
It is only used in the tests, but in two different packages, so it cannot be in a `_test.go` file. Do you think it should go somewhere else?

Reply:
yes, i think it is not a good idea to have a test function in a production code.  IF it is used in 2 packages, we can define it in one package, but in the test file, and use it from there, IMHO

Reply:
Done! PTAL.